README.md

```diff
@@ -20,6 +20,16 @@ And import Penpal into your code with something like:
 
 `import Penpal from 'penpal';`
 
+#### TypeScript
+
+Penpal provides a `types` file for usage with TypeScript; importing is simply the same as above:
+
+`import Penpal from 'penpal';`
+
+If importing with the error codes you will need to do something like:
+
+`import { default as Penpal, ERR_CONNECTION_DESTROYED } from 'penpal';`
```

Comment:
Is your import example here specific to TypeScript users? Typically I would import like this:

`import Penpal, { ERR_CONNECTION_DESTROYED } from 'penpal';`

That kind of sucks if TypeScript users are forced to do it a different way.

Reply:
That’ll probably work too actually. Brain must not have been in gear this morning. I can update in GitHub and update my PR but won’t be at my machine to test the TypeScript until tomorrow

Reply:
No problem. The error constants are also statics on the `Penpal` object, so I don't really expect consumers to import both Penpal and the constants like you've shown here. They should be able to do:

```
import Penpal from 'penpal';
console.log(Penpal.ERR_CONNECTION_DESTROYED);
```

I only export the constants separately in case someone is _only_ using the constants in a particular part of their code and doesn't want to include Penpal in that particular chunk (speaking in Webpack terms). I hope that makes sense. I tried to explain it in the readme:

> This provides an opportunity for build optimization (using tools like Webpack or Rollup) in cases where code only needs access to the error constants and not the rest of Penpal.

Anyway, unless users of TypeScript need to do something unique, I'd probably leave this part out. I really should get a sample app with TypeScript working so I can test this stuff out.

Reply:
I've checked and in typescript we can do `import Penpal, { ERR_CONNECTION_DESTROYED } from 'penpal';` too so I've updated the README